### ISSUE TITLE
bug: failure to parse `esriFieldTypeBlob` (arcpbf)

### ISSUE THREAD

**Issue by JWilliamsonArch** [R-ArcGIS/arcpbf#6]
Hi!

I have noticed that there is an {arcgislayers} bug in arc_select() which prevents it from loading and opening valid ArcGIS Rest Services.

**Description**
The package, {arcgislayers}, cannot load ArcGIS Rest services with the command arc_select() . The rest service is valid and opens with other software.

**Reproducible Example**
```
library(arcgis)
library(tidyverse)
library(sf)

furl <- "https://geonb.snb.ca/arcgis/rest/services/GeoNB_SNB_Municipal_Information/MapServer/1"

Municipal_Roads <- arc_open(furl) %>% 
  arc_select(.)
```
Version:
R version 2024.04.2+764 "Chocolate Cosmos"
Running under: Windows 11 Pro

**Expected behavior**

The features should load into R as sf linestring features.

The features fail to load, and the following error message is printed:
"Error in multi_resp_process_(resps) :                      
  User function panicked: multi_resp_process_"

**Comment by JWilliamsonArch** [comment]
I found that this package could pull down other similar-sized datasets, so the problem does seem to be with the esri blobs as a data type.

I also found that the arcpullr package*  can download this file, even with the Esri blob in place, which is worth mentioning for other people running into this issue.  
* https://cran.r-project.org/web/packages/arcpullr/vignettes/intro_to_arcpullr.html

**Comment by JosiahParry** [comment]
It may also be worth noting that arcpullr is orders of magnitude slower than arcgislayers.
It works because in this feature layer the blobs are entirely `null`—no actual binary data. arcpullr uses json whereas arcpbf uses protocol buffers which are ~1/10th of the memory foot print. They are also processed in Rust.

**Comment by elipousson** [comment]
Stumbled across an example of a blob field in a public FeatureLayer: https://services1.arcgis.com/UWYHeuuJISiGmgXx/arcgis/rest/services/police_CallsForService_PreviousYear/FeatureServer/0

**Comment by JosiahParry** [comment]
Thank you @elipousson ! i'll try exploring these

**Comment by JosiahParry** [comment]
These feature services unfortunately return empty strings. Right now the `blob` branch has handling behavior for these which behaves like so: 

```
``` r
library(arcgislayers)
furl <- "https://services1.arcgis.com/UWYHeuuJISiGmgXx/ArcGIS/rest/services/911_Calls_For_Services_2024/FeatureServer/0"

arc_open(furl) |> 
  arc_select(fields = c("callKey", "HashedRecord"), n_max= 100, crs = 4326) |> 
  head()
#> Blob types not supported.
#> Please report an issue at https://github.com/R-ArcGIS/arcpbf/issues
#> Provide the FeatureService URL if possible
#> Simple feature collection with 6 features and 2 fields (with 6 geometries empty)
#> Geometry type: POINT
#> Dimension:     XY
#> Bounding box:  xmin: NA ymin: NA xmax: NA ymax: NA
#> Geodetic CRS:  WGS 84
#>            callKey HashedRecord    geometry
#> 1 10CC9DA965929F12              POINT EMPTY
#> 2 10CC9DA96592A321              POINT EMPTY
#> 3 10CC9DA96592B7B9              POINT EMPTY
#> 4 10CC9DA96592BCBA              POINT EMPTY
#> 5 10CC9DA96592DFE2              POINT EMPTY
#> 6 10CC9DA96592E370              POINT EMPTY
```


Would this suffice you think?

**Comment by elipousson** [comment]
Yep! I don't need anything in the blob field – I just need the data and the error in the main branch prevents me from accessing these layers. I'm guessing these are created automatically at some point in the process but I don't know much about how these fields work.

**Comment by JosiahParry** [comment]
Nor do I. I haven't found anyone who does either 😅 . I'll merge the branch.

**Comment by JosiahParry** [comment]
Closes by #10
